src/test/java/com/ibanity/apis/client/helpers/IbanityClientSecurityAuthenticationPropertiesKeys.java

```diff
@@ -0,0 +1,9 @@
+package com.ibanity.apis.client.helpers;
+
+public interface IbanityClientSecurityAuthenticationPropertiesKeys {
+
+    String IBANITY_CLIENT_TLS_CERTIFICATE_PATH_PROPERTY_KEY = "client.tls.certificate.path";
+    String IBANITY_CLIENT_TLS_PRIVATE_KEY_PATH_PROPERTY_KEY = "client.tls.private-key.path";
+    String IBANITY_CLIENT_TLS_PRIVATE_KEY_PASSPHRASE_PROPERTY_KEY = "client.tls.private-key.passphrase";
```

Comment:
we need to change "PASSPHRASE", eg "PPHRASE" because this gives a security score "E" on sonarcloud...

Reply:
Why? What's the reason behind that?

Reply:
The system spotted it as a security issue as we could have saved credentials. It wasn't the case but anyway, it's not relevant anymore as constant moved to an interface.